integration-tests/testconfig/testconfig.go

```diff
@@ -433,8 +443,20 @@ root_key_funds_buffer = 1_000
 		}
 	}
 
-	logger.Debug().Msg("Correct test config constructed successfully")
-	return testConfig, nil
+	var customChainSettings []string
+	for _, network := range networks.MustGetSelectedNetworkConfig(c.Network) {
+		if c.NodeConfig != nil && len(c.NodeConfig.ChainConfigTOMLByChainID) > 0 {
+			if _, ok := c.NodeConfig.ChainConfigTOMLByChainID[fmt.Sprint(network.ChainID)]; ok {
+				logger.Warn().Msgf("***** You have provided custom Chainlink Node configuration for network '%s' (chain id: %d). Chainlink Node's default settings won't be used *****", network.Name, network.ChainID)
```

Comment:
We shouldn't be adorning logs with our own decorators. I think it just hurts readability.
```suggestion
				logger.Warn().Msgf("You have provided custom Chainlink Node configuration for network '%s' (chain id: %d). Chainlink Node's default settings won't be used", network.Name, network.ChainID)
```